gui/default/index.html

```diff
@@ -81,6 +81,8 @@
             <li class="divider" aria-hidden="true"></li>
             <li><a href="" ng-click="advanced()"><span class="fas fa-fw fa-cogs"></span>&nbsp;<span translate>Advanced</span></a></li>
             <li><a href="" ng-click="logging.show()"><span class="far fa-fw fa-file-alt"></span>&nbsp;<span translate>Logs</span></a></li>
+            <li class="divider" aria-hidden="true" ng-show="config.gui.debugging"></li>
+            <li><a href="/rest/debug/support" target="_blank" ng-show="config.gui.debugging"><span class="fa fa-user-md"></span>&nbsp;<span translate>Support Bundle</span></li>
```

Comment:
Out of curiosity, how does one enable debugging? I don't think it's available in the UI?

Reply:
I think it refers to  Actions --> Advanced --> GUI --> Debugging

Reply:
I never noticed it, it's not obvious what it does, so perhaps we should add a message explaining what it enables (if it's not already there, have not checked).

Reply:
@AudriusButkevicius I'm not sure are you asking this, just for information.
[Syncthing Docs](https://docs.syncthing.net/users/profiling.html) profiling part explains how to enable `Debugging` and what is for.

Reply:
Essentially, all I was saying is that I never noticed there being a debug flag for GUI available in the UI.

Reply:
I'm not sure where we would expose that. Currently the process is that I say "can you take a profile, here's the process" and link to the doc above which explains it. I envision the new process would be "can you grab a support bundle, here's the process" and a link to that document which would gain a new section.... Or something